### PR TITLE
docs(alerts): Add alerting section to session docs

### PR DESCRIPTION
**Pull request by scttcper**
requires https://github.com/getsentry/sentry-docs/pull/3845

### CODE REVIEW

File: src/docs/sdk/sessions.mdx [hunk 1]
```diff
@@ -308,6 +308,16 @@ with one difference: sessions transitioned to `exited` have their session
 duration averaged. This lets Sentry show you the duration of non crashed
 sessions over time.
 
+## Alerts
+<Alert title="WIP" level="info">
+  Session based alerting is currently in early access.
```
Comment (BYK) [inline src/docs/sdk/sessions.mdx:313]: Does "early access" mean anything in develop docs?

File: src/docs/sdk/sessions.mdx [hunk 2]
```diff
@@ -308,6 +308,16 @@ with one difference: sessions transitioned to `exited` have their session
...
+</Alert>
+
+Trigger an alert when a specified threshold of sessions experience an issue. 
+Create a new [issue alert](https://docs.sentry.io/product/alerts/alert-types/#issue-alerts) and select the "When" condition
```
Comment (BYK) [inline src/docs/sdk/sessions.mdx:317]: Feels like this can use a screenshot?

File: src/docs/sdk/sessions.mdx (hunk 1 above)
Comment (scttcper) [reply src/docs/sdk/sessions.mdx]: removed

File: src/docs/sdk/sessions.mdx (hunk 2 above)
Comment (scttcper) [reply src/docs/sdk/sessions.mdx:314]: added a screenshot

Comment (iProgramStuff) [reply src/docs/sdk/sessions.mdx:314]: I think this is fine, but I kinda feel like the wording can be improved here. 

Maybe something like:
"Trigger an alert when an issue affects a specified percent of sessions. "

**Review by iProgramStuff — APPROVED** [review]

**Pull request merged**